src/renderer/components/side-nav/side-nav.js

```diff
@@ -76,10 +76,6 @@ export default Vue.extend({
   methods: {
     navigate: function (route) {
       this.$router.push('/' + route)
-    },
-
-    goToChannel: function (id) {
```

Comment:
Please double check and see if this can actually be removed. I believe the `@keypress` action is here so that the sideNav works via accessibility controls (Ex: Using Tab to navigate through the menu and hitting enter to navigate to that channel). If it's still needed for this workflow, then please re-add this to the file.